### PR TITLE
Allows for non-lowercase usernames to be verifiable in Hacker News

### PR DESCRIPTION
**Pull request by BrunoBernardino**
Fixes https://github.com/blockstack/blockstack.js/issues/415

### CODE REVIEW

File: dist/blockstack.js [hunk 1]
```diff
@@ -9734,7 +9734,7 @@ module.exports={
   "_args": [
     [
       "bigi@1.4.2",
-      "/Users/larry/git/blockstack.js"
+      "/Users/brunobernardino/Projects/blockstack.js"
```
Comment (BrunoBernardino) [inline dist/blockstack.js]: How can I get rid of these? I `git checkout --`d `docs.json` and `package-lock.json` but apparently that still leaked here. 😬

Comment (BrunoBernardino) [reply dist/blockstack.js]: Ah. It was this: https://github.com/blockstack/blockstack.js/issues/368

**Comment by BrunoBernardino** [comment]
Just noticed this won’t work. The URL can’t be lowercased, because it’s what’s used for the fetching of the content.

I’ll fix it tomorrow.

**Comment by BrunoBernardino** [comment]
Ok, it's ready for a review!

**Comment by larrysalibra** [comment]
Thanks for this @BrunoBernardino - trying this out now!

**Review by larrysalibra — APPROVED** [review]
This works well for me! Thanks for the pull request!

**Pull request merged**